core/main_test.go

```diff
@@ -5,13 +5,32 @@ package autospotting
 
 import (
 	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
 	"reflect"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 )
 
+func TestMain(m *testing.M) {
+	var logOutput io.Writer
```

Comment:
This is the implementation from https://github.com/AutoSpotting/AutoSpotting/pull/354/files#diff-592039bf1534bef34939e90696678d3eR16, minus all the stuff that doesn't make sense before the rest of the changes from #354 are merged.